ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/underlay-overlay/spine-p2p-interfaces.j2

```diff
@@ -42,4 +42,35 @@
 {%             endif %}
 {%         endfor %}
 {%     endfor %}
+{% endfor %}
+{# Point-to-Point Interfaces to Super Spine Switches #}
+  ### p2p Interfaces to Super Spine Switches
+{% set a_spine = namespace() %}
+{% set a_spine.id = spine.nodes[inventory_hostname].id %}
+{# walk over spine ports defined as connections to super-spine #}
```

Comment:
Maybe add a {% if spine['uplinks_to_super_spine_interfaces'] is defined and spine['uplinks_to_super_spine_interfaces'] is not none %} like we have everywhere else?